azure-pipelines.yml

```
@@ -0,0 +1,37 @@
+trigger:
+- master
+
+jobs:
+
+- job: 'Testing_black_Mac'
```

Black is a proper noun, just like Mac and Windows.
```suggestion
- job: 'Testing_Black_Mac'
```